jmx-scraper/src/main/java/io/opentelemetry/contrib/jmxscraper/config/JmxScraperConfig.java

```diff
@@ -76,9 +85,8 @@ public String getServiceUrl() {
     return serviceUrl;
   }
 
-  @Nullable
-  public String getCustomJmxScrapingConfigPath() {
-    return customJmxScrapingConfigPath;
+  public List<String> getJmxConfig() {
+    return jmxConfig;
```

Comment:
I like wrapping with immutable so it's clear that a caller can't change our internal state.
```suggestion
    return Collections.unmodifyableList(jmxConfig);
```

Reply:
Good point, I've applied your suggestion in 911eefe when the attribute is set rather than in the getter and also covered the target system config option.